app/src/main/java/com/example/soccert/ui/databinding/ImageBinding.kt

```diff
@@ -13,3 +14,10 @@ fun loadUrlImageCircle(imageView: ImageView, urlImage: String, cropStatus: Boole
         imageView.loadImageRectangle(urlImage)
     }
 }
+
+@BindingAdapter("app:notificationStatus")
+fun loadStatusNotification(imageView: ImageView, status: String) {
```

Comment:
Em có thể viết dưới dạng extension 
```suggestion
fun ImageView.loadStatusNotification(status: String) {
```
Câu hỏi: vì sao có thể viết bằng 2 cách được ?

Reply:
e thấy khi binding thằng extension nó cũng truyền vào tham số là phần từ đầu tiên luôn, nên cũng giống như thằng binding ạ